eclair-core/src/main/scala/fr/acinq/eclair/io/Monitoring.scala

```diff
@@ -28,9 +28,9 @@ object Monitoring {
 
     val ReconnectionsAttempts = Kamon.counter("reconnections.attempts")
 
-    val OnionMessagesReceived = Kamon.counter("onionmessages.received")
     val OnionMessagesSent = Kamon.counter("onionmessages.sent")
```

Comment:
You're using this metric both for sent and received messages, so it's weird that it's named `Sent`? What about:

```suggestion
    val OnionMessagesProcessed = Kamon.counter("onionmessages.processed")
```

Reply:
Done